lib/reverse_sentence.rb

```diff
@@ -1,6 +1,29 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
+# Time complexity: O(n^2)
 # Space complexity: ?
-def reverse_sentence(my_sentence)
-  raise NotImplementedError
+def reverse_sentence(my_sentence) #using bubble sort method
```

Comment:
This doesn't seem to reverse the sentence, and you don't need a nested loop to reverse something.

I suggest just making 1 loop which goes from 0 to length -1 and swapping elements at index i and index (length - i - 1).